### ISSUE TITLE
Idomeneus

### ISSUE THREAD

**Issue by wehro** [gregorio-project/hyphen-la#55]
*Idomeneus* should be hyphenated `I-do-me-neus` (so far: `Id-o-me-ne-us`). *eu* is a diphthong here and I see no prefix *id* in this name.

**Comment by fradec** [comment]
I agree for the beginning of the word (thank you very much although, it allowed me to delete several faulty and useless patterns!), But for the end of the word, in liturgical context (certainly this word has no chance of appearing there) the final `eus` is necessarily in two syllables, because of the declination.

**Comment by wehro** [comment]
I don't know if it's relevant to liturgical Latin, but the person names ending in *-eus* can have different origins and different syllable boundaries according to their origin.

1. Some names ending in `-æus` or `-ēus` derive from Greek `-αιος` (e. g. *Brisæus/Brisēus*).
2. Some names ending in `-ēus` or `-īus` derive from Greek `-ειος` (e. g. *Dārēus/Dārīus*).
3. Some names ending in `-eus` derive from Greek `-εος` (e. g. *Sōsitheus, Tīmāsitheus*).
4. Many names ending in `-eus` derive from Greek `-ευς` (e. g. *Erechtheus, Idomeneus, Orpheus, Thēseus*).

In the first three cases, there is always a syllabe boundary before `us`.
In the fourth case, `eu` is a diphthong (at least in classical Latin). This case is marked by a tie in several dictionaries: *Erechthe͡us, Idomene͡us, Orphe͡us, Thēse͡us*. The Gaffiot uses `eūs` for the same purpose, as explained [here](https://www.lexilogos.com/latin/gaffiot.php?p=abbreviation).
The problem of the declension of these names is explained as follows by the grammar of Kühner/Holzweissig: Greek endings with diphthong are typically used for the nominative and the vocative (e. g. nom. *Orphe͡us*, voc. *Orphe͡u*), Latin endings without diphthongs for the other cases (e. g. gen. *Orpheī*, dat./abl. *Orpheō*, acc. *Orpheum*).
